index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+ <div class="firstpage">
+
+
+
+<nav>
+    <div class="logo">
+    <a href="">My Logo</a>
+    </div>
+    <aside>
+        <a href="">Portfolio</a>
+        <a href="">About</a>
+        <a href="">Contact</a>
```

Comment:
- [ ] Good job working on this, but this milestone focuses on the mobile version. While you will have the opportunity to design for desktop later, please make sure at the moment it is mobile-focused. Please replace the links with a hamburger as specified in the design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/164293617-65430dda-2d56-4add-93af-75ede787293a.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/164293908-ce077674-b135-4a8a-8c47-22fe8a93b21b.png)